File: Angular/Player/src/app/model/Player.ts
export class Player{

    PlayerId? : number;
    PlayerName? : string ;
    Image? : string ;
    IsAvailable? : boolean;
    BirthDay : Date;
    TeamId? : number;
    TeamName? : string;
    constructor(
    _PlayerId? : number,
    _PlayerName? : string ,
    _Image? : string ,
    _IsAvailable? : boolean,
    _BirthDay?:Date ,
    _TeamId? : number,
    _TeamName? : string
    ){

       this.PlayerId = _PlayerId;
       this.PlayerName = _PlayerName;
       this.Image =_Image;
       this.IsAvailable =_IsAvailable;
       this.BirthDay = _BirthDay;
       this.TeamId = _TeamId;
       this.TeamName = _TeamName;

          
    }

   
}

File: PlayerAngularProject/Model/Player.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace PlayerAngularProject.Model
{
    public class Player
    {
        [Key]
        public int PlayerId { get; set; }
        public string PlayerName { get; set; }
        public string Image { get; set; }
        public bool IsAvailable { get; set; }
        public DateTime BirthDay { get; set; }
        [NotMapped]
        public string TeamName { get; set; }
        [ForeignKey("TeamId")]
        public int TeamId { get; set; }
        public virtual Team Team { get; set; }
    }
}
